vyper/context/types/function.py

```diff
@@ -265,10 +259,7 @@ def from_FunctionDef(
             )
 
         arguments = OrderedDict()
-        if include_defaults:
-            defaults = [None] * (len(node.args.args) - len(node.args.defaults)) + node.args.defaults
-        else:
-            defaults = [None] * len(node.args.args)
+        defaults = [None] * (len(node.args.args) - len(node.args.defaults)) + node.args.defaults
```

Comment:
This is because you should always include the defaults?

Reply:
Yeah, at some point that kwarg stopped being used.  So it's ded code.